azext_edge/edge/params.py

```diff
@@ -1669,6 +1669,30 @@ def load_iotops_arguments(self, _):
             help="Storage account container name where schemas will be stored.",
         )
 
+    with self.argument_context("iot ops schema version") as context:
+        context.argument(
+            "version_name",
+            options_list=["--version", "--ver", "--name", "-n"],
```

Comment:
If we switch to `--version` do we need the `--name` and `-n`? Or is this one of those situations where the version IS the name - just want to make sure we don't block usage of `-n` in the future if we need it

Reply:
`--name`/`-n` could be removed to reduce potential confusion. The PR stems from user confusion with `--name`. What's your take @c-ryan-k .

Reply:
or we can just do both for now